tests/validation/tests/v3_api/common.py

```diff
@@ -2224,4 +2252,55 @@ def login_as_auth_user(username, password):
         'responseType': 'json',
     }, verify=False)
     assert r.status_code in [200, 201]
-    return r.json()['token']
+    return r.json()
+
+
+def auth_get_project():
+    return auth_rbac_data["project"]
+
+
+def auth_get_namespace():
+    return auth_rbac_data["namespace"]
+
+
+def auth_get_user_token(username):
+    if username in auth_rbac_data["users"].keys():
+        return auth_rbac_data["users"][username].token
+    return None
+
+
+def add_role_to_user(user, role):
+    """this function adds a user from the auth provider to given cluster"""
+    admin_client, cluster = get_global_admin_client_and_cluster()
+    project = auth_get_project()
+    ns = auth_get_namespace()
+    if not (project and ns):
+        project, ns = create_project_and_ns(ADMIN_TOKEN, cluster,
+                                            random_test_name("p-test-auth"))
+        auth_rbac_data["project"] = project
+        auth_rbac_data["namespace"] = ns
+    if role in [PROJECT_OWNER, PROJECT_MEMBER, PROJECT_READ_ONLY]:
+        assign_members_to_project(admin_client, user, project, role)
+    else:
+        assign_members_to_cluster(admin_client, user, cluster, role)
+    auth_rbac_data["users"][user.username] = user
+
+
+def auth_resource_cleanup():
+    """ remove the project and namespace created for the AUTH tests"""
+    client, cluster = get_global_admin_client_and_cluster()
+    client.delete(auth_rbac_data["project"])
+    auth_rbac_data["project"] = None
+    auth_rbac_data["ns"] = None
+    crtbs = client.list_cluster_role_template_binding(clusterId=cluster.id)
```

Comment:
Can we list all the cluster role template bindings by  each of the users in auth_rbac_data["users"] and delete them?